### ISSUE TITLE
Filtering on Cohort Exit

### ISSUE THREAD

**Issue by ericaVoss** [OHDSI/Atlas#1171]
### Expected behavior
You can generate a cohort exist data and then filter it (e.g. I only want people who were exposed to this injectable this long given a persistence window of 90 days and surveillance window of 30).

### Actual behavior
There is no filter for cohort end date (or not one I know about)

### Steps to reproduce behavior
N/A

**Comment by chrisknoll** [comment]
Is what you're asking for essentially a 'minimum cohort duration' that a person must have in order to be included in the cohort?.  

Example, Person A has a single exposure lasting 1 day, with your surveillance window of 30, the person would exit the cohort after 30 days from exposure, leading to a cohort duration of 31 days.

Person B had 2 exposures, each lasting 1 day, on day 1 and day 20, these exposure would be chained together, with the exit being defined on day 51 (30 days after the end of the second exposure which happened on day 20).

Then, the new feature would say 'exclude episodes with less than 45 days, dropping person A but keeping person B.

Is that use-case accurate for what you want?

**Comment by ericaVoss** [comment]
Yes - I'm asking for minimum cohort duration to help select a cohort.  Specifically I want to find persistent users of a drug.

Yes - per your example if we want to exclude <45 days, this filter would allow me to drop Person A and keep Person B.

That all sounds good.

**Comment by chrisknoll** [comment]
Ok, i just wanted to clarify the use-case because it doesn't have to be confined to drug cohorts. Whichever way the episodes are constructed (ex: chaining together condition occurrences and only keep episodes that last longer than 90 days), in the end we want to say 'drop episodes that do not meet a minimum duration'.

**Comment by ericaVoss** [comment]
Agree, doesn't have to be confined to drug cohorts - whichever way the episodes are constructed.

And filters should be the regular ones ATLAS uses:
<365
>=365
>365
BETWEEN 10 AND 20
ETC

**Comment by pbr6cornell** [comment]
this is related to idea of allowing multiple cohort exit criteria, not only one.
